Readme.md

```diff
@@ -415,7 +415,21 @@ options: {
 		// ...
 	}
 }
-````
+```
+
+#### customOutputs
+
+Type: `array` Default: `undefined`
+
+Allows for custom content to be generated and output in the same way as `htmlDemo`.
+
+Each entry in `customOutputs` should be an object with two parameters:
+
+* `template` - (`string`) the path to the underscore-template you wish to use.
+* `dest` - (`string`) the path to the destination where you want the resulting file to live.
+* `context` \[optional\] - (`object`) a hash of values to pass into the context of the template at compile-time
+
+At compile-time each template will have access to the same context as the compile-time environment of `htmlDemoTemplate` (as extended by the `context` object, if provided. See config-example below.
```

Comment:
Is it really compile-time? Not run time? Compilation of templates usually means compilation to JS function that you could call later.